### PR TITLE
Update link to tdd-exercise to be github repo

### PR DESCRIPTION
**Pull request by yangashley**

### CODE REVIEW

File: intro-to-tests/tdd-exercise.instructor.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+## We are no longer using replit for livecodes and activities
 Replit Activity: [https://replit.com/@adadev/tdd-exercise#README.md](https://replit.com/@adadev/tdd-exercise#README.md)
 
 Replit Solution: [https://replit.com/@adadev/tdd-exercise-solution](https://replit.com/@adadev/tdd-exercise-solution)
 
 Old Repo: [https://github.com/AdaGold/tdd-exercise](https://github.com/AdaGold/tdd-exercise), not used because students haven't learned how to collaborate in Git yet.
 
+Current Repo (introduced in C22): [https://github.com/Ada-Activities/tdd-exercise](https://github.com/Ada-Activities/tdd-exercise)
+
 Solution Branch in Old Repo: [https://github.com/AdaGold/tdd-exercise/blob/solution-python/tests/test_blackjack_score.py](https://github.com/AdaGold/tdd-exercise/blob/solution-python/tests/test_blackjack_score.py)
```
Comment (kelsey-steven-ada) [inline intro-to-tests/tdd-exercise.instructor.md:12]: How do you feel about moving the current link to the top of the page so it's the first thing folks see and removing the AdaGold repo links?

```suggestion
Current Repo (introduced in C22): [https://github.com/Ada-Activities/tdd-exercise](https://github.com/Ada-Activities/tdd-exercise)
- Solution is on the `solution` branch in this repo

## We are no longer using replit for livecodes and activities
Replit Activity: [https://replit.com/@adadev/tdd-exercise#README.md](https://replit.com/@adadev/tdd-exercise#README.md)

Replit Solution: [https://replit.com/@adadev/tdd-exercise-solution](https://replit.com/@adadev/tdd-exercise-solution)
```

File: intro-to-tests/tdd-exercise.instructor.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 
 The TDD Exercise is introduced as part of the problem set (Part 1) and parts 2 and 3 are meant to be a classroom activity. This classroom activity likely needs 2 hours.
 
 Many folks are not familiar with BlackJack. Hopefully by providing the prompt as part of the problem set students will have time to review the rules.
-    - If not knowing the rules of blackjack proves to be an impediment to the learning goals, we should probably replace the prompt for this exercise.
+    - If not knowing the rules of blackjack proves to be an impediment to the learning goals, we should probably replace the prompt for this exercise.
```
Comment (kelsey-steven-ada) [inline intro-to-tests/tdd-exercise.instructor.md:17]: I'm sorry if I'm overlooking it, but I can't tell what changed on this line, what was altered?

**Review by kelsey-steven-ada — APPROVED** [review]
I think it looks good as-is, but left a suggestion for cleaning up some of the old links in case we want to remove some noise from the instructor doc.

File: intro-to-tests/tdd-exercise.instructor.md (hunk 2 above)
Comment (yangashley) [reply intro-to-tests/tdd-exercise.instructor.md:17]: I don't think anything changed except for the line number cuz I added a new line. Not sure why it's highlighted

**Pull request merged**